android/source/VideoLocker/src/org/edx/mobile/social/SocialFactory.java

```diff
@@ -12,9 +13,19 @@
     
     public static ISocial getInstance(Activity activity, int type) {
         if (type == TYPE_GOOGLE) {
-            return new GoogleOauth2(activity);
+            if (Config.getInstance().getThirdPartyTraffic().isGoogleEnabled()) {
```

Comment:
Ideally the social code wouldn't even get called if it's disabled, but I guess that would be too invasive a change for now?